src/main/java/capstone/mfslbackend/model/FantasyWeek.java

```diff
@@ -31,8 +32,19 @@ public class FantasyWeek {
     @ToString.Exclude
     @JsonIgnore
     @ManyToOne(optional = false)
-    @JoinColumn(name = "fantasy_team_id", nullable = false)
-    private FantasyTeam fantasyTeam;
+    @JoinColumn(name = "fantasy_team_a_id", nullable = false)
+    private FantasyTeam fantasyTeamA;
+
+    @ToString.Exclude
+    @OneToOne(orphanRemoval = true)
+    @JoinColumn(name = "fantasy_week_id")
+    private FantasyWeek fantasyWeek;
```

Comment:
This association is not needed. We never use it. I think you added it when we weren't sure how we were going to implement fantasyWeek